lib/Star/Kata/Model/Kata.php

```diff
@@ -97,7 +97,7 @@ public function start()
             $step->init();
         }
 
-        return true;
+        return new StartedKata($this);
```

Comment:
Are you using the `State` pattern here to prevent users from calling invalid methods or operations?